packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -21,27 +22,75 @@ declare module 'slate' {
   }
 }
 
-// Define our app...
-const RichTextEditor = () => {
+const toContentfulDocumentDebounced = (document, schema) => {
+  // TODO: Do we need to debounce 500ms, like in master?
```

Comment:
I'm seeking ideas from reviewers here. We debounce the conversion in `master` by 500ms, not sure if we need to be doing this anymore due to ShareJS removal

Reply:
If we are not sure then let's maintain the same behaviour as in `master` for now until we actually investigate the reason behind it a bit more.

Reply:
yeah, makes sense 👍